datacapture/src/main/java/com/google/android/fhir/datacapture/extensions/MoreQuestionnaireItemComponents.kt

```
@@ -581,6 +581,15 @@ internal val Questionnaire.QuestionnaireItemComponent.unitOption: List<Coding>
     return this.extension
       .filter { it.url == EXTENSION_QUESTIONNAIRE_UNIT_OPTION_URL }
       .map { it.value as Coding }
+      .ifEmpty {
+        // https://build.fhir.org/ig/HL7/sdc/behavior.html#initial
```

i think there're two things in the specification that are open to interpretation:

1. do we consider multiple intial values? what if multiple initial values have the same unit? i think in this case we should deduplicate. please do that in the code.
2. the text in sdc ig only talks about the case where the initial values don't have the numerical value. but what happens if the initial values do have numerical values as well as unit? in this case, it does seem to make sense to use the units as well so that at least these initial values can be represented.
3. what if the initial values include units that do not appear in the unit option extension? again, i feel like maybe in this case the easiest thing to do is to include the units in the initial values. yes, we can throw an exception, but that seems very unforgiving.

==> if we consider point 2 and 3 above together, perhaps an easier way to write the code is just to join the unit options with all the units in the initial values and then deduplicate them. can you do that?